src/osd/OSD.cc

```diff
@@ -4258,27 +4258,44 @@ PerfCounters* OSD::create_recoverystate_perf()
 
 int OSD::shutdown()
 {
+  // vstart overwrites osd_fast_shutdown value in the conf file -> force the value here!
```

Comment:
Suspicious comments. Is there something missing in this PR?
If this is for testing purposes, then it should be a separate commit.